File: graphql/src/main/kotlin/com/example/graphql/config/TestConfig.java
package com.example.graphql.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import java.time.Duration;

/**
 * @Auth yeyu
 * @Date 2024/11/21
 */
@Configuration
@ConfigurationProperties(prefix = "test")
@Data
public class TestConfig {
    private Duration outTime;
}

File: webSocket/src/main/kotlin/com/example/webSocket/config/TestConfig.kt
package com.example.webSocket.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration
import kotlin.time.Duration

/**
 *
 *@Auth yeyu
 *@Date 2024/11/21
 *
 */
@ConfigurationProperties(prefix = "test")
@Configuration
class TestConfig {
    var outTime:Duration = Duration.ZERO
}